### PR TITLE
Move include files from internal folders to master/includes folder (Part 2)

### LINKED ISSUE #2179: Move include files from internal folders to the master\includes folder.
We need to remove include files from internal (primarily in C# and VB) and move them all here: https://github.com/dotnet/docs/tree/master/includes
 
If the file already exist in the main includes folder, we will keep the original file and delete the other instead of moving it.

If a file is moved, we will do the following:

1. Remove the metadata inside that looks like this:

translation.priority.ht: 
  - "cs-cz"
  - "de-de"
  - "es-es"
  - "fr-fr"
  - "it-it"
  - "ja-jp"
  - "ko-kr"
  - "pl-pl"
  - "pt-br"
  - "ru-ru"
  - "tr-tr"
  - "zh-cn"
  - "zh-tw"

2.  Fix all links to the file, using the format ~/includes/tokenname.md
 
Suggest reviewers
@mairaw

### PR DESCRIPTION
**Pull request by tompratt-AQ**
# Title
Move include files from internal folders to master/includes folder (Part 2)

## Summary
This PR completes the job that was mostly done by PR #2187 by moving the include files from these internal directories:

- docs/csharp/programming-guide/interop/includes
- docs/csharp/programming-guide/types/includes
- docs/visualbasic/developing-apps/customizing-extending-my/includes
- docs/visualbasic/developing-apps/includes
- docs/visualbasic/developing-apps/printing/includes
- docs/visualbasic/language-reference/objects/includes
- docs/visualbasic/misc/includes
- docs/visualbasic/reference/command-line-compiler/includes  

Once this PR has been reviewed and merged, all include files will be in the (~/includes) folder and will be named with dashes instead of underscores. This will fix Issue 2179.

Fixes #2179 

## Suggested Reviewers
@mairaw

### CODE REVIEW

File: docs/visual-basic/programming-guide/program-structure/limitations.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ translation.priority.ht:
   - "zh-tw"
 ---
 # Visual Basic Limitations
-Earlier versions of [!INCLUDE[vbprvb](~/includes/vbprvb-md.md)] enforced boundaries in code, such as the length of variable names, the number of variables allowed in modules, and module size. In [!INCLUDE[vbprvblong](../../../visual-basic/developing-apps/customizing-extending-my/includes/vbprvblong_md.md)], these restrictions have been relaxed, giving you greater freedom in writing and arranging your code.  
+Earlier versions of [!INCLUDE[vbprvb](~/includes/vbprvb-md.md)] enforced boundaries in code, such as the length of variable names, the number of variables allowed in modules, and module size. In [!INCLUDE[vbprvblong](~/includes/vbprvblong-md.md)], these restrictions have been relaxed, giving you greater freedom in writing and arranging your code.
```
Comment (rpetrusha) [inline docs/visual-basic/programming-guide/program-structure/limitations.md]: In [token] --> In Visual Basic .NET

File: docs/visual-basic/reference/command-line-compiler/compiler-options-listed-alphabetically.md [hunk 2]
```diff
@@ -64,7 +64,7 @@ The [!INCLUDE[vbprvb](~/includes/vbprvb-md.md)] command-line compiler is provide
 |[/main](../../../visual-basic/reference/command-line-compiler/main.md)|Specifies the class that contains the `Sub``Main` procedure to use at startup.|
```
Comment (rpetrusha) [inline docs/visual-basic/reference/command-line-compiler/compiler-options-listed-alphabetically.md]: Can you fix the formatting here, @tompratt-AQ? 
`Sub``Main` --> `Sub Main`

File: docs/visual-basic/reference/command-line-compiler/how-to-invoke-the-command-line-compiler.md [hunk 3]
```diff
@@ -57,7 +57,7 @@ You can invoke the command-line compiler by typing the name of its executable fi
   
 1.  Use the Windows Search feature to find Vbc.exe on your local disk.  
   
-     The exact name of the directory where the compiler is located depends on the location of the Windows directory and the version of the [!INCLUDE[Compact](../../../visual-basic/reference/command-line-compiler/includes/compact_md.md)] installed. If you have more than one version of the [!INCLUDE[Compact](../../../visual-basic/reference/command-line-compiler/includes/compact_md.md)] installed, you must determine which version to use (typically the latest version).  
+     The exact name of the directory where the compiler is located depends on the location of the Windows directory and the version of the [!INCLUDE[Compact](~/includes/compact-md.md)] installed. If you have more than one version of the [!INCLUDE[Compact](~/includes/compact-md.md)] installed, you must determine which version to use (typically the latest version).
```
Comment (rpetrusha) [inline docs/visual-basic/reference/command-line-compiler/how-to-invoke-the-command-line-compiler.md]: Wow. This is even the wrong token, and it's been here for more than a decade. Change both occurrences of [token] to ".NET Framework".

File: docs/visual-basic/reference/command-line-compiler/netcf.md [hunk 4]
```diff
@@ -104,7 +104,7 @@ Sets the compiler to target the [!INCLUDE[Compact](../../../visual-basic/referen
     -   `Line`  
   
 ## Example  
- The following code compiles `Myfile.vb` with the [!INCLUDE[Compact](../../../visual-basic/reference/command-line-compiler/includes/compact_md.md)], using the versions of Mscorlib.dll and Microsoft.VisualBasic.dll found in the default installation directory of the [!INCLUDE[Compact](../../../visual-basic/reference/command-line-compiler/includes/compact_md.md)] on the C drive. Typically, you would use the most recent version of the [!INCLUDE[Compact](../../../visual-basic/reference/command-line-compiler/includes/compact_md.md)].  
+ The following code compiles `Myfile.vb` with the [!INCLUDE[Compact](~/includes/compact-md.md)], using the versions of Mscorlib.dll and Microsoft.VisualBasic.dll found in the default installation directory of the [!INCLUDE[Compact](~/includes/compact-md.md)] on the C drive. Typically, you would use the most recent version of the [!INCLUDE[Compact](~/includes/compact-md.md)].
```
Comment (rpetrusha) [inline docs/visual-basic/reference/command-line-compiler/netcf.md]: Mscorlib.dll --> mscorlib.dll (the first "m" is lowercase)

**Review by rpetrusha — CHANGES_REQUESTED** [review]
@tompratt-AQ, unfortunately, some of the Visual Basic 2005 content tied general features to the specific Visual Basic 2005 version. Otherwise, the PR looked really good. There are also some merge conflicts that need to be resolved, though.

**Comment by tompratt-AQ** [comment]
@rpetrusha Thanks for detail on each correction! I made those changes and a change for the merge conflict.

**Review by rpetrusha — APPROVED** [review]

**Comment by rpetrusha** [comment]
LGTM. I'll merge.

**Pull request merged**